gradle.properties

```diff
@@ -1 +1,6 @@
-version=1.0.4-SNAPSHOT
\ No newline at end of file
+version=1.0.4-SNAPSHOT
+kotlinVersion=1.8.22
+kolasuVersion=1.5.45
+lsp4jVersion=0.21.1
+luceneVersion=9.8.0
+junitVersion=5.7.1
```

Comment:
I think centralizing all the versions makes sense.
Can we take the chance to centralize the versions of plugins such as ktlint or kotlin jvm as well?

Reply:
Done, introduced version catalogs